File: VoiceTypingUniversal/.build/arm64-apple-macosx/release/ZipArchive.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* ZipArchive_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"ZipArchive_ZipArchive.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/Volumes/Awaye-SSD/AntiGravity/STT/VoiceTypingUniversal/.build/arm64-apple-macosx/release/ZipArchive_ZipArchive.bundle"];
    }

    return preferredBundle;
}

File: VoiceTypingSwift/.build/arm64-apple-macosx/release/VoiceTyping.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("VoiceTyping_VoiceTyping.bundle").path
        let buildPath = "/Volumes/Awaye-SSD/AntiGravity/STT/VoiceTypingSwift/.build/arm64-apple-macosx/release/VoiceTyping_VoiceTyping.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}